react/week4/github-userSearch/src/components/Form.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from "react";
+
+function UserSearch({ handleSearch }) {
+	const [searchUser, setSearchUser] = useState("");
+	handleSearch(searchUser);

```

Comment:
We should use `useEffect()` here, otherwise we will call `handleSearch()` 2x times. In this case it doesn't have side effects, but it could in different cases.
```suggestion
	useEffect(() => {
		handleSearch(searchUser);
	}, [searchUser]);

```